ARGO/src/ForceComponent.cpp

```diff
@@ -2,15 +2,17 @@
 #include "ForceComponent.h"
 
 ForceComponent::ForceComponent() :
```

Comment:
do you need this anymore. You can just set the values to have default values like you have for has friction in the overloaded constructor